public/css/form.css

```diff
@@ -0,0 +1,93 @@
+/* General Styles */
+body {
+    background-image: url("https://images.unsplash.com/photo-1542435503-956c469947f6?q=80&w=1974&auto=format&fit=crop&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D"); 
```

Comment:
what will happen if the image link not works?

Reply:
If the image is not fetched, no error will be thrown. The background will turn white by default. However we can keep a fall back color in case the background image does not work.